packages/transitive-overlay/src/util.ts

```diff
@@ -177,6 +177,27 @@ function makeStop(stop: Place, coordinate?: number[]) {
     : stop;
 }
 
+/**
+ * Helper function that clones the 'to' stop and applies flex logic
+ */
+function cloneToStop(leg: Leg): Place {
+  const toStop = { ...leg.to };
```

Comment:
We don't know that `leg.to` will be a stop! Let's make that clear in the variable name

Reply:
I renamed it to _flexSupportStop_ in my soon to be pushed commit!